src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -1699,6 +1699,27 @@ export default class Aircraft {
     }
 
     /**
+     * Checks if the aircraft is inside the airspace of a specified airport
+     *
+     * @for AircraftInstanceModel
+     * @method isInsideAirspace
+     * @param  {airport} airport the airport whose airspace we are checking
+     * @return {Boolean}
+     * @private
+     */
+    isInsideAirspace(airport) {
+        let inside = false;
+
+        if (airport.perimeter) {    // polygonal airspace boundary
+            inside = point_in_area(this.position, airport.perimeter);
+        } else {    // simple circular airspace boundary
+            inside = this.distance <= airport.ctr_radius && this.altitude <= airport.ctr_ceiling;
```

Comment:
why not initialize `inside` with this condition and then only update `inside` if `airport.perimeter`?  This has the same effect as the if/else block, but its a little cleaner because now there is no else block.